chart/compass/values.yaml

```diff
@@ -1060,6 +1060,7 @@ global:
         - "application_template:write"
         - "formation_template:read"
         - "formation_template:write"
+        - "application.local_tenant_id:write"
```

Comment:
This would allow anyone to provide local tenant id to any system which means that one can register a system and steal another customer's system. A major security issue from my PoV which was the reason for this scope.

Reply:
Yeah, we figured that out as well. We'll now merge it.